composed_api.rst

```diff
@@ -665,121 +1003,268 @@ Object Model
+
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| Key               | Value Type   | Value Description                                                                   |
++===================+==============+=====================================================================================+
+| datatype          | string       | The datatype (e.g., LAS 1.2, DTM).                                                  |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| name              | string       | The name of the product.                                                            |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| pk                | integer      | The primary key of the product.                                                     |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| sensor            | string       | The sensor used to make the collection.                                             |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| collect\_at       | timestamp    | The date of collection. ISO 8601 format as UTC.                                     |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| classification    | string       | The security classification.                                                        |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| area              | float        | The area of the geometry in sq\_km.                                                 |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| filesize          | integer      | The size of the product on the filesystem in bytes.                                 |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| point\_count      | integer      | The total number of points in the product.                                          |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| density           | float        | The average point density of the product.                                           |
++-------------------+--------------+-------------------------------------------------------------------------------------+
+| coverage\_ratio   | string       | The percent of the product area covered by the AOI. Only available in AOI detail.   |
```

Comment:
String seems an odd choice for this. It shows up several places. I'd prefer a float in the range [0.0, 1.0]. Also, `percent_coverage` may be more meaningful.